Configuration/StandardSequences/python/DigiToRaw_Repack_cff.py

```diff
@@ -60,24 +60,19 @@
 from EventFilter.SiStripRawToDigi.SiStripDigis_cfi import siStripDigis
 siStripDigisHLT = siStripDigis.clone(ProductLabel = "rawDataRepacker")
 
-from RecoLocalTracker.Configuration.RecoLocalTracker_cff import siStripZeroSuppression
-siStripZeroSuppressionHLT = siStripZeroSuppression.clone(
-    RawDigiProducersList =cms.VInputTag(cms.InputTag("siStripDigisHLT","VirginRaw"), cms.InputTag("siStripDigisHLT","ProcessedRaw"), cms.InputTag("siStripDigisHLT","ScopeMode")
-    ))
-    
+from RecoLocalTracker.Configuration.RecoLocalTracker_cff import siStripZeroSuppressionHLT
+
 from RecoLocalTracker.SiStripClusterizer.DefaultClusterizer_cff import *
 siStripClustersHLT = cms.EDProducer("SiStripClusterizer",
                                     Clusterizer = DefaultClusterizer,
                                     DigiProducersList = cms.VInputTag(
-                                        cms.InputTag('siStripDigisHLT','ZeroSuppressed'),
-                                        cms.InputTag('siStripZeroSuppressionHLT','VirginRaw'),
-                                        cms.InputTag('siStripZeroSuppressionHLT','ProcessedRaw'),
-                                        cms.InputTag('siStripZeroSuppressionHLT','ScopeMode')),
+                                        ('siStripDigisHLT','ZeroSuppressed'),
+                                        ('siStripZeroSuppressionHLT','VirginRaw'),
+                                        ('siStripZeroSuppressionHLT','ProcessedRaw'),
+                                        ('siStripZeroSuppressionHLT','ScopeMode')),
                                 )
```

Comment:
```suggestion
                                        ('siStripZeroSuppressionHLT','ScopeMode')) ]
```

Reply:
TypeError: DigiProducersList does not already exist, so it can only be set to a CMS python configuration type

Reply:
Ah, ok: [this](https://github.com/cms-sw/cmssw/pull/38423/files/ee76098c510b6ca6bb076146d46297f6f66c010a..fe658676d03835ff49f7164bfc0dc19aa1d6c16d#diff-3732e87bbaf24e365fc0ebffb06a04c9504db622c2d77522e3a27bc16f4720aaR69-R72) was not the change I asked in my review, and that's why I got confused. I thought that this fix from your last commit was referring to https://github.com/cms-sw/cmssw/pull/38423#discussion_r909325862

Reply:
I might have gotten myself a bit lost.  Did I address all your comments with the latest commit?